crytic_compile/platform/solc_standard_json.py

```diff
@@ -290,21 +290,11 @@ def parse_standard_json_output(
             for contract_name, info in file_contracts.items():
                 # for solc < 0.4.10 we cant retrieve the filename from the ast
                 if skip_filename:
-                    filename = convert_filename(
-                        file_path,
-                        relative_to_short,
-                        compilation_unit.crytic_compile,
-                        working_dir=solc_working_dir,
-                    )
+                    filename = compilation_unit.crytic_compile.filename_lookup(file_path)
                 else:
-                    filename = convert_filename(
-                        file_path,
-                        relative_to_short,
-                        compilation_unit.crytic_compile,
-                        working_dir=solc_working_dir,
-                    )
```

Comment:
I don't know what purposed `skip_filename` serves. It seems like it hasn't done anything since https://github.com/crytic/crytic-compile/commit/8eec1421df9feb9f185704f3a339c77ad6eac830#diff-5919c9835671c24ce72c5fecf7f8dec4bfbc62baf8e06c70d227ffd013629e38L158-L174